examples/hellohttp/e2e-test.sh

```diff
@@ -60,7 +60,7 @@ check_msg() {
    # Wait some more for after IP address allocation for the service to come
    # alive
    echo "Got IP Adress! Sleeping 30s more for service to come alive..."
```

Comment:
change the comment in echo as well?

Reply:
Done locally. Will push once/if CI passes